src/operator/every.ts

```diff
@@ -5,6 +5,25 @@ import { Subscriber } from '../Subscriber';
 
 /**
  * Returns an Observable that emits whether or not every item of the source satisfies the condition specified.
+ *
+ * @example <caption>A simple example emitting true if all elements are less than 5, false otherwise</caption>
+ *  Observable.of(1, 2, 3, 4, 5, 6)
+ *     .every(x => x < 5)
+ *     .subscribe(x => console.log(x)); // -> false
+ * 
+ * @example <caption>An example passing this (if you cannot use ES6 arrow function)</caption>
+ * class MyClass {
```

Comment:
I think we can ditch the `class` here and just use a function and the observable.